packages/apollos-data-connector-postgres/src/media/migrations/002_update_media_primary_keys.js

```diff
@@ -0,0 +1,34 @@
+import { Sequelize } from 'sequelize';
+
+async function up({ context: queryInterface }) {
+  await queryInterface.sequelize.transaction(async (t) => {
+    await queryInterface.addColumn(
+      'media',
+      'originId',
+      {
+        primaryKey: true,
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      { transaction: t }
+    );
+    await queryInterface.addColumn(
+      'media',
+      'originType',
+      { type: Sequelize.STRING, allowNull: false },
+      { transaction: t }
+    );
+
+    await queryInterface.addIndex('media', ['originId', 'originType'], {
+      unique: true,
+    });
+  });
+}
+
+async function down({ context: queryInterface }) {
+  await queryInterface.dropTable('media');
+}
+
+const name = '002-update_media_primary_keys';
+
+module.exports = { up, down, name, order: 4 };
```

Comment:
So that it's one bigger than `create media` :) 
```suggestion
module.exports = { up, down, name, order: 5 };
```